dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNDataInterfaceIIDM.cpp

```diff
@@ -199,8 +202,6 @@ DataInterfaceIIDM::getBusName(const std::string& componentName, const std::strin
         }
         break;
       }
-      case ComponentInterface::UNKNOWN:
```

Comment:
Removing this breaks the build

Reply:
@e2l-su Please put back this part of the code as removing it breaks the compilation

Reply:
done :) Sorry, I thought at first it was some mystake. How did you see "it breaks" ? How come it works fine locally ?